mock-core-configs/etc/mock/templates/fedora-eln.tpl

```diff
@@ -38,36 +38,71 @@ cost=2000
 enabled=0
 skip_if_unavailable=False
 
-[eln]
-name=Fedora ELN - Developmental modular packages for the next Enterprise Linux release
+[eln-baseos]
+name=Fedora - ELN BaseOS - Developmental packages for the next Enterprise Linux release
+baseurl=https://odcs.fedoraproject.org/composes/production/latest-Fedora-ELN/compose/BaseOS/$basearch/os/
 #metalink=https://mirrors.fedoraproject.org/metalink?repo=eln&arch=$basearch
-baseurl=https://odcs.fedoraproject.org/composes/production/latest-Fedora-ELN/compose/Everything/$basearch/os/
 enabled=1
 repo_gpgcheck=0
 type=rpm
 gpgcheck=1
-gpgkey=file:///usr/share/distribution-gpg-keys/fedora/RPM-GPG-KEY-fedora-rawhide-primary
+gpgkey=file:///etc/pki/rpm-gpg/RPM-GPG-KEY-fedora-eln-$basearch
 skip_if_unavailable=False
 
-[eln-debuginfo]
-name=Fedora ELN - Debug
-metalink=https://mirrors.fedoraproject.org/metalink?repo=eln-debug&arch=$basearch
-#baseurl=https://odcs.fedoraproject.org/composes/production/latest-Fedora-ELN/compose/Everything/$basearch/debug/tree
+[eln-baseos-debuginfo]
+name=Fedora - ELN BaseOS - Debug
+baseurl=https://odcs.fedoraproject.org/composes/production/latest-Fedora-ELN/compose/BaseOS/$basearch/debug/tree
+#metalink=https://mirrors.fedoraproject.org/metalink?repo=eln-debug&arch=$basearch
 enabled=0
 repo_gpgcheck=0
 type=rpm
 gpgcheck=1
-gpgkey=file:///usr/share/distribution-gpg-keys/fedora/RPM-GPG-KEY-fedora-rawhide-primary
+gpgkey=file:///etc/pki/rpm-gpg/RPM-GPG-KEY-fedora-eln-$basearch
 skip_if_unavailable=False
 
-[eln-source]
-name=Fedora ELN - Source
-metalink=https://mirrors.fedoraproject.org/metalink?repo=eln-source&arch=$basearch
-#baseurl=https://odcs.fedoraproject.org/composes/production/latest-Fedora-ELN/compose/Everything/source/tree/
+[eln-baseos-source]
+name=Fedora - ELN BaseOS - Source
+baseurl=https://odcs.fedoraproject.org/composes/production/latest-Fedora-ELN/compose/BaseOS/source/tree/
+#metalink=https://mirrors.fedoraproject.org/metalink?repo=eln-source&arch=$basearch
 enabled=0
 repo_gpgcheck=0
```

Comment:
Is it already signed? Can we enable the gpgcheck here?

Reply:
@xsuchy It doesn't appear to be supported. If it's OK with you, I'll drop `repo_gpgcheck` from the per-repo configs so it will inherit whatever is in `[main]` or the default.

Reply:
Then just leave it here. It is always good to be explicit.